### DISCUSSION TITLE
How can I browse the preview window with the keyboard

### DISCUSSION THREAD

**Discussion by Leiyi548** [JetBrains/ideavim#377] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76139422/137256001-ade00c18-636f-4a2c-aa19-632d1b3ec5ce.png)
How to use the keyboard to move him is a problem. I looked for the settings and found no solution.

**Comment by AlexPl292** [comment, marked as answer]
Hi, unfortunately, IdeaVim now works only in the editor and the project tree. So, this is not possible at the moment.

**Reply by Leiyi548** [reply]
Thank you

**Reply by Josh-Piper** [reply]
Is this possible currently? Would be nice to C-w o to pop up's and previews
